portability-types-common/src/main/java/org/datatransferproject/types/common/models/calendar/RecurrenceRule.java

```diff
@@ -1,5 +1,6 @@
 package org.datatransferproject.types.common.models.calendar;
 
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
```

Comment:
no idea if this matters, but this file change is only a single import line added, not used

Reply:
Thanks! Removed